codec-http2/src/test/java/io/netty5/handler/codec/http2/Http2FrameCodecTest.java

```diff
@@ -423,6 +423,7 @@ public void unknownFrameTypeOnConnectionStream() throws Exception {
 
     @Test
     public void unknownFrameOnMissingStream() throws Exception {
+        setUp();
```

Comment:
?

Reply:
The tests all call the method which sets up the class, including the `frameInboundWriter` which was null causing the test to error out.